isomorphe/templates/fragments/transform_results_preview.html.j2

```diff
@@ -0,0 +1,90 @@
+<div class="fr-table"> <!-- fr-table--no-caption fr-table--no-scroll -->
```

Comment:
@abulte I couldn't figure out how to limit DSFR table height :(

We can't paginate (because we might want to add item-level selection, which doesn't work across pages), and I'd rather not display a 1000-line table. Especially since "Suivant" is at the bottom of the page (so different from select preview, but maybe we should standardize?)

I can limit the entire table height but can't get something working with sticky headers (= only scroll contents) that doesn't end up having independent layouts for thead/tbody (so columns don't align).

Any hints? If that even possible... :pray:

Reply:
I'm afraid this is one of those "you shouldn't do that" situation :p Hijacking the scroll is dangerous and difficult, especially in the middle of a page.

The only case where it's "acceptable", and not too hard to do, is on a full screen table and a sticky content on top (cf explore.data.gouv.fr). Also cf the data view on a resource of data.gouv.fr where they paginate.

I'd look into:
- Moving the "Suivant" at the top, I don't think it's a big deal since we have a use case for it. Repeat it at the end.
- Doesn't datatable handle the pagination/selection you want?

Reply:
I was trying to avoid adding datatables, and if we don't need per-item selection I think we can do without with the 1st option.

We'd have something like:

![image](https://github.com/user-attachments/assets/8b0e25c7-8c7f-497b-8a59-316c4ea6c010)

Not the best layout IMO but it works well enough.